sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/contacts/ContactDirectorySteps.java

```diff
@@ -231,6 +236,30 @@ public ContactDirectorySteps(
           webDriverHelpers.clickWebElementByText(INDOORS_OPTION, indoorsOption);
           webDriverHelpers.waitForPageLoadingSpinnerToDisappear(40);
         });
+    And(
+        "I click on All button in Contact Directory Page",
+        () -> {
+          webDriverHelpers.clickOnWebElementBySelector(ALLBUTTON_CONTACT);
+          TimeUnit.SECONDS.sleep(8); // needed for table refresh
+        });
+    And(
+        "I click on Converted to case pending button on Contact Directory Page",
+        () -> {
+          webDriverHelpers.clickOnWebElementBySelector(CONVERTED_TO_CASE_BUTTON);
+          TimeUnit.SECONDS.sleep(8); // needed for table refresh
+        });
+    And(
+        "I click on Active contact button in Contact Directory Page",
+        () -> {
+          webDriverHelpers.clickOnWebElementBySelector(ACTIVE_CONTACT_BUTTON);
+          TimeUnit.SECONDS.sleep(8); // needed for table refresh
```

Comment:
change it with this: waitUntilAListOfWebElementsAreNotEmpty

Reply:
I changed it, but this test is not written by myself